.github/workflows/build-pr.yml

```diff
@@ -42,8 +47,18 @@ jobs:
           echo "BRANCH_NAME=pr-releases/pr-${PR_NUMBER}" >> $GITHUB_ENV
           echo "COMMIT_HASH=$(git rev-parse HEAD)" >> $GITHUB_ENV
 
-      - name: Post comment on PR
+      - name: Find Previous Comment
+        uses: peter-evans/find-comment@v2
```

Comment:
I've looked through the code in this action - it's very simple and we could implement it ourselves if this becomes an issue.